src/Components/Intercom.js

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+import * as Scrivito from 'scrivito';
+import Helmet from 'react-helmet';
+
+class Intercom extends React.Component {
+  static getIntercomAppId() {
+    const rootPage = Scrivito.Obj.root();
+
+    if (!rootPage) {
+      return;
+    }
+
+    return rootPage.get('intercomAppId');
+  }
+
+  static bindFakeIntercom() {
```

Comment:
As discussed, we shouldn't name it "fake". The "snippet API" doesn't indicate this.
* [ ] find a better name.